analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -56,11 +57,19 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
             new InitializationVectorShouldBeRandom(),
             new HashesShouldHaveUnpredictableSalt());
 
-        protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules { get; } = ImmutableDictionary<DiagnosticDescriptor, RuleFactory>.Empty
+        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabled)
+        {
```

Comment:
The `AlwaysEnabled` will already use the Roslyn on ITs and Peach, right? We don't want that yet.

Reply:
> The AlwaysEnabled will already use the Roslyn on ITs and Peach, right? We don't want that yet.

If we turn it off, S222 is not run with the current implementation (I suppose).